test/middleware/authentication.middleware.unit.ts

```diff
@@ -29,4 +30,12 @@ describe("authentication middleware tests", () => {
     expect(mockAuthMiddleware).toHaveBeenCalledWith(expectedAuthMiddlewareConfig);
     expect(mockAuthReturnedFunction).toHaveBeenCalledWith(req, res, next);
   });
+
+  it("should not call authentication middleware when loading accessibility page", () => {
+    req.originalUrl = CONFIRMATION_STATEMENT + ACCESSIBILITY_STATEMENT;
+    authenticationMiddleware(req, res, next);
+
+    expect(mockAuthMiddleware).toBeCalled;
```

Comment:
this expect isn't really doing anything.

I would set the test up similarly to the test in test/routes/routes.spec.unit.ts

But expect the response.text not to contain "sign in"

That way the test is asserting that if the url is whitelisted, it does not redirect them to the sign in page